frontend/src/layout/panel-layout/ProjectTree/ProjectTree.tsx

```diff
@@ -582,7 +589,26 @@ export function ProjectTree({
                     (root === 'shortcuts://' && item.record?.href && item.record.href.split('/').length - 1 === 1)
 
                 if (showProductMenuItems) {
-                    if (item.name === 'Product analytics') {
+                    if (item.tags && item.tags.length > 0) {
+                        const tag = item.tags[0]
+                        return (
+                            <ButtonPrimitive
+                                iconOnly
+                                isSideActionRight
+                                className="z-2 group justify-end hover:justify-center"
```

Comment:
@adamb70 @corywatilo can I get some help here? Do you think this should be right-aligned like this or center-aligned? CSS is heavily simplified if we center align it.

Context here: https://www.loom.com/share/5497d7a2048f4ac3a7f87ade2b6bbc4c